### PR TITLE
(issue 872): implemented isscalar() function

### PR DESCRIPTION
**Pull request by alixdamman**

### CODE REVIEW

File: larray/core/ufuncs.py [hunk 1]
```diff
@@ -302,3 +303,36 @@ def broadcastify(func):
...
+    Returns
+    -------
+    bool
+        `True` if `element` is a scalar type, `False` if it is not.
+
+    Examples
+    --------
+    >>> isscalar(3.1)
+    True
+    >>> isscalar(np.array(3.1))
```
Comment (gdementen) [inline larray/core/ufuncs.py:326]: I wouldn't show this in the doctest (too obscure) for our users. On the other hand, a test with an **larray** scalar array could be interesting mostly to avoid having to add the information about that corner case to the description or the "Returns" paragraph **but** I would move it much lower in the list, because it is a corner case.

File: larray/core/ufuncs.py [hunk 2]
```diff
@@ -302,3 +303,36 @@ def broadcastify(func):
      a0   6   2   5   0
      a1   5   2   3   0
 """
+
+
+def isscalar(element: Any) -> bool:
```
Comment (gdementen) [inline larray/core/ufuncs.py:308]: is isscalar an ufunc (numpy universal function)? I don't think it is, so moving this to another module would be a good idea I think.

Comment (alixdamman) [reply larray/core/ufuncs.py:308]: Correct. I move it to util/misc.py or do I create a new module in the `core` directory?

Comment (gdementen) [reply larray/core/ufuncs.py:308]: AFAIR, util/misc.py are for internal functions not api-exposed functions, so I wouldn't put it there. Maybe core/misc.py?

Comment (alixdamman) [reply larray/core/ufuncs.py:308]: OK for core/misc.py

**Comment by gdementen** [comment]
this PR LGTM, what's blocking it?
